core/iwasm/aot/aot_runtime.c

```diff
@@ -519,55 +496,19 @@ memory_instantiate(AOTModuleInstance *module_inst, AOTModuleInstance *parent,
                 module->aux_stack_size);
     LOG_VERBOSE("  heap offset: %u, heap size: %d\n", heap_offset, heap_size);
 
-    memory_data_size = (uint64)num_bytes_per_page * init_page_count;
     max_memory_data_size = (uint64)num_bytes_per_page * max_page_count;
     bh_assert(memory_data_size <= UINT32_MAX);
```

Comment:
memory_data_size isn't set, bh_assert may fail in debug mode

Reply:
right, deleted the assert as we already have the same one when the value is actually set in `wasm_allocate_linear_memory`